sandbox/plc4py/src/main/python/org/apache/plc4x/PlcDriverManager.py

```diff
@@ -18,6 +18,7 @@
 import subprocess
 import time
 import warnings
+from contextlib import contextmanager
 
 from generated.org.apache.plc4x.interop.InteropServer import Client, PlcException
```

Comment:
@JulianFeinauer can you explain the import? I can't find the `generated.org...` package nowhere in the project and I am wondering what I've missed... I was able to build the project but some `target` directories appeared.

I had this info during the build:
```
-- Building without tests
-- Found FLEX: /usr/bin/flex (found version "2.6.4")
-- Found BISON: /usr/bin/bison (found version "3.0.4")
-- ----------------------------------------------------------
-- Thrift version:                               0.13.0 (0.13.0)
-- Thrift package version:                       0.13.0
--
-- Build configuration summary
--   Build compiler:                             ON
--   Build libraries:                            ON
--   Build tests:                                OFF
--   Build type:                                 RelWithDebInfo
--
-- Language libraries:
--
--   Build as3 library:                          OFF
--    - Adobe Flex compc was not found - did you set env var FLEX_HOME?
--
--   Build C++ library:                          OFF
--
--   Build C (GLib) library:                     OFF
--    - GLib missing
--
--   Build Java library:                         OFF
--
--   Build Python library:                       OFF
--    - Python libraries missing
--
--   Build Haskell library:                      OFF
--    - GHC missing
--    - Cabal missing
--
-- ----------------------------------------------------------
-- Configuring done
-- Generating done
```

But I'm not sure if this is somehow related (especially the Haskell part :D ).